zip-codes/scripts/map.js

```diff
@@ -7,6 +7,17 @@ function initMap() {
     scrollwheel: true,
     zoom: 8
   });
-
+  if (zips.length) {
```

Comment:
Where is zips coming from? You don't have a variable named zips defined in this file :)